templates/openstack/managed.html

```diff
@@ -12,7 +12,7 @@ <h1>Managed private cloud</h1>
     <h2 class="p-heading--3">Cut your infrastructure costs with Managed OpenStack</h2>
     <p>Like a public cloud, but more cost-effective. Deployed and managed 24x7 exclusively for you.</p>
     <p>Offload from your operations team and accelerate your time-to-market with Canonical's Managed OpenStack.</p>
-    <p><a href="/openstack/managed#get-in-touch" class="p-button--positive u-no-margin--left js-invoke-modal is-inline">Get Managed OpenStack</a></p>
+    <p><a href="/openstack/managed#get-in-touch" data-testid="interactive-form" class="p-button--positive u-no-margin--left js-invoke-modal is-inline">Get Managed OpenStack</a></p>
```

Comment:
I think that using a more specific test id would be better for clarity (e.g. `interactive-form-link`). When reading `findByTestId("interactive-form")` in a test it may suggest that it's a test id for a form itself, not to a link.